diff --git a/node_modules/react-native-reanimated/lib/module/animation/util.js b/node_modules/react-native-reanimated/lib/module/animation/util.js
index 80aa4e3..59a74f4 100644
--- a/node_modules/react-native-reanimated/lib/module/animation/util.js
+++ b/node_modules/react-native-reanimated/lib/module/animation/util.js
@@ -37,9 +37,6 @@ export function isValidLayoutAnimationProp(prop) {
 
   return prop in LAYOUT_ANIMATION_SUPPORTED_PROPS;
 }
-if (__DEV__ && ReducedMotionManager.jsValue) {
-  logger.warn(`Reduced motion setting is enabled on this device. This warning is visible only in the development mode. Some animations will be disabled by default. You can override the behavior for individual animations, see https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#reduced-motion-setting-is-enabled-on-this-device.`);
-}
 export function assertEasingIsWorklet(easing) {
   'worklet';
 
diff --git a/node_modules/react-native-reanimated/src/animation/util.ts b/node_modules/react-native-reanimated/src/animation/util.ts
index 2466ac9..aff4d54 100644
--- a/node_modules/react-native-reanimated/src/animation/util.ts
+++ b/node_modules/react-native-reanimated/src/animation/util.ts
@@ -75,12 +75,6 @@ export function isValidLayoutAnimationProp(prop: string) {
   return (prop as LayoutAnimationProp) in LAYOUT_ANIMATION_SUPPORTED_PROPS;
 }
 
-if (__DEV__ && ReducedMotionManager.jsValue) {
-  logger.warn(
-    `Reduced motion setting is enabled on this device. This warning is visible only in the development mode. Some animations will be disabled by default. You can override the behavior for individual animations, see https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#reduced-motion-setting-is-enabled-on-this-device.`
-  );
-}
-
 export function assertEasingIsWorklet(
   easing: EasingFunction | EasingFunctionFactory
 ): void {
